tests/x11/firefox/firefox_extensions.pm

```diff
@@ -26,17 +26,21 @@ sub run {
     send_key "ctrl-shift-a";
     assert_screen('firefox-addons_manager', 90);
     assert_and_click "firefox-extensions";
-    assert_and_click "firefox-searchall-addon";
-    type_string "flagfox\n";
-    assert_and_click('firefox-extensions-flagfox');
-    assert_and_click('firefox-extensions-add-to-firefox');
-    wait_still_screen 6;
-    send_key 'alt-a';
-    # close the flagfox relase notes tab and flagfox search tab
-    send_key_until_needlematch 'firefox-addons-plugins', 'ctrl-w', 3, 3;
-    # refresh the page to see addon buttons
-    send_key 'f5';
-    assert_screen('firefox-extensions-flagfox_installed', 90);
+    for (1 .. 5) {
+        assert_and_click 'firefox-searchall-addon';
+        type_string "flagfox\n";
+        assert_and_click 'firefox-extensions-flagfox';
+        wait_still_screen 3;
```

Comment:
```suggestion
```

Reply:
yes, because of failure, it's better to wait :wink: